examples/3.x_api/pytorch/nlp/huggingface_models/language-modeling/quantization/static_quant/pt2e/run_benchmark.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+set -x
+
+function main {
+
+  init_params "$@"
+  run_benchmark
+
+}
+
+# init params
+function init_params {
+  for var in "$@"
+  do
+    case $var in
+      --topology=*)
+          topology=$(echo $var |cut -f2 -d=)
+      ;;
+      --dataset_location=*)
+          dataset_location=$(echo $var |cut -f2 -d=)
+      ;;
+      --input_model=*)
+          input_model=$(echo $var |cut -f2 -d=)
+      ;;
+       --output_model=*)
+           tuned_checkpoint=$(echo $var |cut -f2 -d=)
+       ;;
+      *)
+          echo "Error: No such parameter: ${var}"
+          exit 1
+      ;;
+    esac
+  done
+
+}
+
+# run_tuning
+function run_benchmark {
+
+    if [ "${topology}" = "opt_125m_pt2e_static" ]; then
+        model_name_or_path="facebook/opt-125m"
+        output_dir="saved_results"
+    fi
+    python run_clm_no_trainer.py --model ${model_name_or_path} --accuracy --output_dir ${output_dir} --tasks "lambada_openai"
```

Comment:
we should support accuracy/performance mode for pt2e right? not only accuracy. Like what we did for fx, https://github.com/intel/neural-compressor/blob/437c8e75706cff1767dcde115e428654766b3f18/examples/pytorch/image_recognition/torchvision_models/quantization/ptq/cpu/fx/run_benchmark.sh#L28